tools/nameid/nameid.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/letsencrypt/boulder/issuance"
+)
+
+func usage() {
+	fmt.Printf("Usage: %s [OPTIONS] cert1.pem certN.pem certN+1.pem\n", os.Args[0])
```

Comment:
Ultra-nit:

```suggestion
	fmt.Printf("Usage: %s [OPTIONS] cert1.pem ... certN.pem\n", os.Args[0])
```

Or, 

```suggestion
	fmt.Printf("Usage: %s [OPTIONS] [CERTIFICATES]\n", os.Args[0])
```

Also, the usage function should print the flags (https://pkg.go.dev/flag#PrintDefaults).

And we should document (in usage and the README) that this expects issuer certificates, not end-entity certificates (particularly since the README mentions that nameid can come from end-entity certificates).

Reply:
The readme text comes straight from [here](https://github.com/letsencrypt/boulder/blob/861ea99cc72b9b3d114dc0630d85e8ebe1005896/issuance/issuer.go#L27-L31), should that be changed too?

Reply:
No, the README text is good. I just mean that in the context of a tool for printing nameids, if you say they can come from either an issuer or an end-entity, it's important to clarify that for _this_ tool they need to come from an issuer.